const socketCatchAsync = (fn) => {
  return async (socket, io, payload) => {
    try {
      return await fn(socket, io, payload);
    } catch (error) {
      console.log("ðŸ”ŒSocket errorðŸ”Œ", error);
    }
  };
};

module.exports = socketCatchAsync;
